exampleCourse/questions/template/figure/dynamic/server.py

```diff
@@ -0,0 +1,57 @@
+import io
+import random
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+def generate(data):
+    # Select a random shape.
+    shape = random.choice(["diamond", "circle"])
+    data["params"]["shape"] = shape
+
+    # Select a random radius.
+    radius = random.choice([1, 2, 3])
```

Comment:
Nit: Question is more dynamic if the random radius is chosen from a range rather than just one of three numbers: https://numpy.org/doc/stable/reference/random/generated/numpy.random.randint.html

Reply:
Done in 66e121f.